docs/database/README.md

```diff
@@ -0,0 +1,4 @@
+# BridgeDb database structure
+
+These creation statements were extracted from a Derby file.
+They may not correspond exactly to the original structure.
```

Comment:
Can you clarify here if this SQL also applies to the MySQL backend?

Reply:
It does work, I removed the indexes though.

Reply:
What do you mean with "it does work"? So, this schema is identical for Derby and MySQL?

Reply:
I can't tell if it's similar, in the sense that I don't have the original one.
This schema **is** compatible with MySQL. I tested it. I removed the indexes though. So I didn't test with the indexes.